docs/os/boot-process/cloud-init/index.md

```diff
@@ -0,0 +1,22 @@
+---
+title: Cloud-init
+layout: os-default
+
+---
+
+## Cloud-init
+
+Userdata and metadata can be fetched from a cloud provider, VM runtime, or management service during the RancherOS boot process. Since v0.8.0, this process occurs while RancherOS is still running from memory and before System Docker starts. It is configured by the `rancher.cloud_init.datasources` configuration parameter. For cloud-provider specific images, such as AWS and GCE, the datasource is pre-configured.
```

Comment:
need to tell the user what "Userdata" and "metadata" are, how they're different, and how the might be used.